crates/iota-graphql-rpc/src/server/builder.rs

```diff
@@ -450,6 +440,14 @@ impl ServerBuilder {
         )
         .map_err(|e| Error::ServerInit(format!("Failed to create pg connection pool: {e}")))?;
 
+        {
```

Comment:
Isn't this the place where we should skip migration consistency check?

```suggestion
        if !config.connection.skip_migration_consistency_check {
```

Reply:
Very good catch, that was the very point of this change 😅 
Fixed in [d7b4d01](https://github.com/iotaledger/iota/pull/9261/commits/d7b4d01f9e6c2020fa97b319cf195f4ba0ca7d1c)